### ISSUE TITLE
How to change dropout rate at runtime

### ISSUE THREAD

**Issue by paolorota** [tensorflow/tensorflow#33716]
I am trying to switch to tf2.0. From the documentation of Dropout it is possible to set dropout rate in the declaration. When I call the dropout operation it does not seem possible to change the dropout rateo anymore. 

Suppose I would like to soften my dropout rate along training how can I do? Is there a walkaround?

```
class mymodel(Model):
    def __init__(self, units=128):
        super(mymodel, self).__init__()
        self.conv1_1 = tf.keras.layers.Conv2D(filters=units, kernel_size=3, strides=1)
        self.dropout1 = tf.keras.layers.Dropout(0.5)

        self.fc = tf.keras.layers.Dense(units=10)


    def call(self, x, training=False):

        x = tf.pad(x, paddings=[[0, 0], [1, 1], [1, 1], [0, 0]])
        x = self.conv1_1(x, training=training)
        x = tf.keras.activations.relu(x)
        x = self.dropout1(x) # how to change the rateo?
        x = self.fc(x)
```

**Comment by jvishnuvardhan** [comment]
@paolorota This is not Build/Installation or Bug/Performance issue. Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. GitHub is mainly for addressing bugs in installation and performance. Thanks!
